doc/consumer-manager.rst

```diff
@@ -25,9 +25,27 @@ Create a JMS Serializer
 -----------------------
 
 In order to deserialize the payload of an AMQP message we have to create a Serializer object
-The easiest way to do so is by using the ``SerializerBuilder`` from this library::
+The easiest way to do so is by using the ``SerializerBuilder`` from the JMS library::
 
-    $serializer = JMS\Serializer\SerializerBuilder::create()->build();
+    use Rebuy\Amqp\Consumer\Serializer\JMSSerializerAdapter;
+    use JMS\Serializer\SerializerBuilder;
+
+    $serializer = SerializerBuilder::create()->build();
+    $serializerAdapter = new JMSSerializerAdapter($serializer);
+
+If you'd rather want to use the symfony serializer, do the following::
+
+    use Rebuy\Amqp\Consumer\Serializer\SymfonySerializerAdapter;
+    use Symfony\Component\Serializer\Serializer;
+    use Symfony\Component\Serializer\Encoder\XmlEncoder;
+    use Symfony\Component\Serializer\Encoder\JsonEncoder;
+    use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+
+    $encoders = array(new XmlEncoder(), new JsonEncoder());
```

Comment:
Short array syntax maybe? Does this library have a minimum required PHP version what should be mentioned?

Reply:
5.6 is the current minimum, will update the syntax, thx :)